understanding/concurrent-input-mechanisms.html

```diff
@@ -58,128 +56,98 @@
+               <p>デバイスが主たる入力メカニズムを備えていても、利用者はデバイスとやりとりするときに、代替の入力メカニズムの使用を選択してもよい。例えば、スマートフォン及びタブレットにおける主たる入力メカニズムはタッチスクリーンである。これらのデバイスの利用者は、タッチスクリーンを使用する代わりに、ペアリングされたマウス又は外付けキーボードを使用することを選択してもよい。
                </p>
-               <p>Users should be able to switch input mechanisms at any point should the user determine
-                  that certain tasks and interactions are more easily accomplished by using an alternative
-                  input mechanism. Content must not limit the user's interaction to any particular input
-                  mechanism unless the restriction is essential, or is required to ensure the security
-                  of the content or to respect user settings.
+               <p>特定のタスク及びインタラクションが代替の入力メカニズムを使用することでより簡単に遂行できると利用者が判断した場合、利用者はいつでも入力メカニズムを切り替えることができるべきである。コンテンツは、利用者によるインタラクションを、特定の入力メカニズムの使用に限定してはならない。ただし、その制限が必要不可欠である場合、すなわちコンテンツのセキュリティを確保する、又は利用者による設定を尊重する必要がある場合を除く。
                </p>
-               <p>Note: A touch-typing web application, which teaches users how to touch-type on a keyboard
-                  and/or measures their proficiency and speed, would be an example of an essential limitation
-                  to a particular input mechanism. 
+               <p>注記: 利用者がキーボード上でどのようにタッチタイピングするかを教える、及び／又はその習熟度及び速度を測定する、タッチタイピング練習ウェブアプリケーションは、特定の入力メカニズムに対する必要不可欠な制限の例である。
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   					
-                  <li>Users can interact with web content with whichever input mechanism is preferred and
-                     available to them.
+                  <li>利用者は、好みの入力メカニズムを使用してウェブコンテンツとやりとりができる。
                   </li>
                   
-                  <li>Users may switch between input mechanisms when they desire or the circumstances require
-                     it.
+                  <li>利用者は、自らの意図又は状況の求めに応じて、入力メカニズムを切り替えることができる。
                   </li>
                   
-                  <li>Users are allowed to add and remove input mechanisms at any point, where supported
-                     by the operating system.
+                  <li>利用者は、オペレーティングシステムがサポートしていれば、任意で入力メカニズムを追加及び削除することができる。
                   </li>
                   				
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A user with mobility impairment pairs a mouse and keyboard to her mobile phone with
-                     a touchscreen. The phone can thereafter be operated by those input devices and the
-                     content does not accept the touchscreen as the only input mechanism.
+                  <li>運動障害のある利用者は、マウス及びキーボードを、タッチスクリーン付きスマートフォンにペアリングする。これにより、スマートフォンはそれらの入力デバイスによっても操作することができるようになり、コンテンツはタッチスクリーンを唯一の入力メカニズムとして受け入れなくなる。
```

Comment:
「タッチスクリーンを唯一の入力メカニズムとして受け入れなくなる」は「タッチスクリーンを受け入れなくなる」と読めて混乱を招きそうです。たとえばですが、 accept をもう少しぼんやりと訳してみるのはどうでしょうか。

```suggestion
                  <li>運動障害のある利用者は、マウス及びキーボードを、タッチスクリーン付きスマートフォンにペアリングする。これにより、スマートフォンはそれらの入力デバイスによっても操作することができるようになり、コンテンツがタッチスクリーンを唯一の入力メカニズムとすることはなくなる。
```